src/Nncase.Importer/TFLite/TFLiteImporter.cs

```diff
@@ -338,6 +338,16 @@ private void Visit(in tflite.Operator op)
             // tflite.BuiltinOperator.ZEROS_LIKE,
             _ => UnSupportedOp(builtinCode.ToString()),
         };
+
+        List<string> outputsNames = new();
+
+        for (int i = 0; i < op.GetOutputsArray().Length; i++)
```

Comment:
Assign `op.GetOutputsArray().Length` to a variable to avoid calls at each iteration.